data/summary.go

```diff
@@ -0,0 +1,19 @@
+package data
+
+import (
+    "time"
+    "go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type Summary struct {
+    ID                 primitive.ObjectID  `json:"-" bson:"_id,omitempty"`
+	UserID             string              `json:"UserID" bson:"_userId"`
+
+	LastUpdated        time.Time   `json:"lastUpdated" bson:"lastUpdated"`
+	LastData           time.Time   `json:"lastData" bson:"lastData"`
```

Comment:
Based on reading the code, it looks like `lastData` will be end time. Since we spend a fair bit of effort figuring what is the start time, consider exposing it here since AFAICT it is not (necessarily) same as `lastUpload`.